app/models/content_item.rb

```diff
@@ -28,6 +28,15 @@ class ContentItem < ActiveRecord::Base
   NON_RENDERABLE_FORMATS = %w(redirect gone).freeze
   EMPTY_BASE_PATH_FORMATS = %w(government).freeze
 
+  has_one :state
+  has_one :location
+  has_one :translation
+  has_one :user_facing_version
+
```

Comment:
I don't want us to make this sort of decision in the context of a single change.  The current structure isn't there due to ignorance of ActiveRecord, it's there for a reason.

If we want to change approach it's something we should get together and discuss in a wider context.